CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/MapViewTileRenderer.java

```diff
@@ -461,17 +464,46 @@ public void mousePressed(final MouseEvent event) {
     @Override
     public void mouseDragged(final MouseEvent event) {
         assert !SwingUtilities.isEventDispatchThread();
-
-        if (event.getButton() == PConstants.CENTER) {
-            // zoom to box
-            boxZoomEnabled = true;
-            boxDeltaX = event.getX();
-            boxDeltaY = event.getY();
-        } else if (event.getButton() == PConstants.RIGHT) {
-            // select markers
-            boxSelectionEnabled = true;
-            boxDeltaX = event.getX();
-            boxDeltaY = event.getY();
+        
+        //Checks to see if click was on the map
+        final boolean ignoreMapInteractions = isIgnoreMapInteractions();
+        
+        switch (event.getButton()) {
+            case PConstants.CENTER:
+                // zoom to box
+                boxZoomEnabled = true;
+                boxDeltaX = event.getX();
+                boxDeltaY = event.getY();
+                break;
+            case PConstants.RIGHT:
+                // select markers
+                boxSelectionEnabled = false;
+                
+                if (!ignoreMapInteractions) {
+                    // Triggers a pan event for a right click & drag if within map
+                    dispatcher.register(map, PanMapEvent.TYPE_PAN, map.getId());
+                    Location oldLocation = map.getLocation(pmouseX, pmouseY);
+                    Location newLocation = map.getLocation(mouseX, mouseY);
+                    PanMapEvent panMapEvent = new PanMapEvent(this, map.getId(), PanMapEvent.PAN_BY);
+                    panMapEvent.setFromLocation(oldLocation);
+                    panMapEvent.setToLocation(newLocation);
+                    dispatcher.fireMapEvent(panMapEvent);
+                    dispatcher.unregister(map, PanMapEvent.TYPE_PAN, map.getId());
+                }   
+                break;
+            case PConstants.LEFT:
+                // select markers
+                final boolean isMeasuring = overlays.stream().filter( overlay -> 
+                        ToolsOverlay.class.isInstance(overlay) && overlay.isEnabled()
```

Comment:
There should not be any references to specific plugins in the tile renderer if at all possible. Perhaps instead you could add an abstract method to `MapOverlay` to signal when the overlay has priority on interactions. This way any overlay could take control of interactions when it makes sense for that overlay.